ts/kpt-functions/src/types.ts

```diff
@@ -166,6 +166,29 @@ export class Configs {
     return this.functionConfig;
   }
 
+  /**
+   * Returns the functionConfig data field if defined.
+   */
+  getFunctionConfigDataOrThrow() {
```

Comment:
rename to getFunctionConfigMap.

So we have the following methods:

getFunctionConfig
getFunctionConfigMap
getFunctionConfigValue
getFunctionConfigValueorThrow

Reply:
Done